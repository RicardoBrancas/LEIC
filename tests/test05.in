s
m
l
a A #computer would deserve to be called #intelligent if it could deceive a human into believing that it was human. #turing
s
m
l
a The first principle is that you must not fool #yourself and you are the easiest person to fool. #feynman
s
m
l
a Prediction is very difficult, especially about the #future. #Bohr
s
m
l
a #Science is a differential equation. #Religion is a boundary condition. #Turing
s
m
l
a A man provided with paper, pencil, and rubber, and subject to strict discipline, is in effect a #universal #machine. #turing
s
s
m
l
a A #computer would deserve to be called #intelligent if it could deceive a human into believing that it was human. #turing
s
m
l
a The first principle is that you must not fool #yourself and you are the easiest person to fool. #feynman
s
m
l
a Prediction is very difficult, especially about the #future. #Bohr
s
m
l
a #Science is a differential equation. #Religion is a boundary condition. #Turing
s
m
l
a A man provided with paper, pencil, and rubber, and subject to strict discipline, is in effect a #universal #machine. #turing
s
m
l
a A #computer would deserve to be called #intelligent if it could deceive a human into believing that it was human. #turing
s
m
l
a The first principle is that you must not fool #yourself and you are the easiest person to fool. #feynman
s
m
l
a Prediction is very difficult, especially about the #future. #Bohr
s
m
l
a #Science is a differential equation. #Religion is a boundary condition. #Turing
s
m
l
a A man provided with paper, pencil, and rubber, and subject to strict discipline, is in effect a #universal #machine. #turing
s
s
m
l
a A #computer would deserve to be called #intelligent if it could deceive a human into believing that it was human. #turing
s
m
l
a The first principle is that you must not fool #yourself and you are the easiest person to fool. #feynman
a Prediction is very difficult, especially about the #future. #Bohr
a #Science is a differential equation. #Religion is a boundary condition. #Turing
a A man provided with paper, pencil, and rubber, and subject to strict discipline, is in effect a #universal #machine. #turing
s
x
