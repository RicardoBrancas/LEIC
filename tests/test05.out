0 0
3 3
#computer 1
#computer 1
#intelligent 1
#turing 1
5 5
#computer 1
#computer 1
#feynman 1
#intelligent 1
#turing 1
#yourself 1
7 7
#bohr 1
#bohr 1
#computer 1
#feynman 1
#future 1
#intelligent 1
#turing 1
#yourself 1
9 10
#turing 2
#turing 2
#bohr 1
#computer 1
#feynman 1
#future 1
#intelligent 1
#religion 1
#science 1
#yourself 1
11 13
11 13
#turing 3
#turing 3
#bohr 1
#computer 1
#feynman 1
#future 1
#intelligent 1
#machine 1
#religion 1
#science 1
#universal 1
#yourself 1
11 16
#turing 4
#turing 4
#computer 2
#intelligent 2
#bohr 1
#feynman 1
#future 1
#machine 1
#religion 1
#science 1
#universal 1
#yourself 1
11 18
#turing 4
#turing 4
#computer 2
#feynman 2
#intelligent 2
#yourself 2
#bohr 1
#future 1
#machine 1
#religion 1
#science 1
#universal 1
11 20
#turing 4
#turing 4
#bohr 2
#computer 2
#feynman 2
#future 2
#intelligent 2
#yourself 2
#machine 1
#religion 1
#science 1
#universal 1
11 23
#turing 5
#turing 5
#bohr 2
#computer 2
#feynman 2
#future 2
#intelligent 2
#religion 2
#science 2
#yourself 2
#machine 1
#universal 1
11 26
#turing 6
#turing 6
#bohr 2
#computer 2
#feynman 2
#future 2
#intelligent 2
#machine 2
#religion 2
#science 2
#universal 2
#yourself 2
11 29
#turing 7
#turing 7
#computer 3
#intelligent 3
#bohr 2
#feynman 2
#future 2
#machine 2
#religion 2
#science 2
#universal 2
#yourself 2
11 31
#turing 7
#turing 7
#computer 3
#feynman 3
#intelligent 3
#yourself 3
#bohr 2
#future 2
#machine 2
#religion 2
#science 2
#universal 2
11 33
#turing 7
#turing 7
#bohr 3
#computer 3
#feynman 3
#future 3
#intelligent 3
#yourself 3
#machine 2
#religion 2
#science 2
#universal 2
11 36
#turing 8
#turing 8
#bohr 3
#computer 3
#feynman 3
#future 3
#intelligent 3
#religion 3
#science 3
#yourself 3
#machine 2
#universal 2
11 39
11 39
#turing 9
#turing 9
#bohr 3
#computer 3
#feynman 3
#future 3
#intelligent 3
#machine 3
#religion 3
#science 3
#universal 3
#yourself 3
11 42
#turing 10
#turing 10
#computer 4
#intelligent 4
#bohr 3
#feynman 3
#future 3
#machine 3
#religion 3
#science 3
#universal 3
#yourself 3
11 52
